tests/unit/executor/test_organize.py

```diff
@@ -220,6 +220,7 @@ def _organize_and_assert(
             base_dir=base_dir,
             overwrite=overwrite,
         )
+        expected = cast(List[Tuple[List[str], str]], expected)
```

Comment:
It seems that `expected` declared as `List[Any]` in the function declaration is also bad because it can be an exception too?